package.json

```diff
@@ -11,8 +12,16 @@
     "gulp-livereload": "~2.1.0",
     "gulp-plumber": "~0.6.3",
     "gulp-serve": "~0.2.0",
+    "insert-css": "^0.2.0",
     "jquery": "^2.1.3",
     "rimraf": "~2.2.8",
     "vinyl-source-stream": "^1.0.0"
+  },
+  "main": "src/index.coffee",
+  "browserify": {
+    "transform": ["coffeeify", "brfs"]
+  },
+  "scripts": {
+    "preview" : "browserify src/index.coffee -o ./preview/boxbits.js"
```

Comment:
I guess the question is more about what will be used to compile coffescript and put the result into `./dist` folder. If we use npm/browserify internally I'd rather we use it throughout the whole project and keep gulp's role to minimum. Or remove gulp wholesale and have `"prerelease"` npm script that will be something like `browserify -t coffeeify brfs index.coffee > ./dist/boxbits.js` and that will be main entry for bower.json or any other registry.